ORStools/common/isochrones_core.py

```diff
@@ -27,6 +27,9 @@
  ***************************************************************************/
 """
 
+from typing import Any, Generator
+
+from qgis._core import QgsMapLayer
```

Comment:
```suggestion
from qgis.core import QgsMapLayer
```

Pycharm tends to do this when you use the helper to add the import, but the _ classes are an internal qgis detail